src/Multiselect.jsx

```diff
@@ -118,9 +118,9 @@ var Multiselect = React.createClass({
   },
 
   getInitialState(){
-    var { data, value, valueField, searchTerm } = this.props
+    var { value, valueField, searchTerm } = this.props
       , dataItems = splat(value).map( item => dataItem(data, item, valueField))
-      , data = this.process(data, dataItems, searchTerm)
+      , data = this.process(this.props.data, dataItems, searchTerm)
```

Comment:
this is gonna break here, `data` is used on the line above, i'd keep it and use processedData as the new var